css/index.css

```diff
@@ -126,6 +155,103 @@ p {
     padding-right: 0;
 }
 
+#box1 {
+    background-color: teal;
+}
+
+#box2 {
+    background-color: gold;
+}
+
+#box3 {
+    background-color: cadetblue;
+}
+
+#box4 {
+    background-color: coral;
+}
+
+#box5 {
+    background-color: crimson;
+}
+
+#box6 {
+    background-color: forestgreen;
+}
+
+#box7 {
+    background-color: darkorchid;
+}
+
+#box8 {
+    background-color: hotpink;
+}
+
+#box9 {
+    background-color: indigo;
+}
+
+#box10 {
+    background-color: dodgerblue;
+}
+
+.about {
+   display: flex;
+   flex-direction: column;
+}
+
+.about p {
+    padding-bottom: 2%;
+}
+
+.dotted-line {
+    border: 1px dashed black;
+    width: 800px;
+    margin: auto;
+}
+
+.section {
+    display: flex;
+    flex-flow: row wrap;
+}
+
+.clicks {
+    display: flex;
+    flex-flow: column wrap;
+    height: 400px;
+    width: 400px;  
+}
+
+.clicks img, h2, p {
+    margin: 3px;
+}
+
+.clicks2 {
+    display: flex;
+    flex-flow: column wrap;
+    height: 400px;
+    width: 400px;
+}
+
+.clicks2 img, h2, p {
+    margin: 3px;
+}
+
+.dotted-line2 {
+    border: 1px dashed black;
+    width: 390px;
+    margin: auto;
+}
+
+button {
+    border: 2px solid black;
+    background-color: white;
+    padding: 2%;
+    margin: 2%;
+    width: 100px;
+    border-radius: 10px;
+}
+
```

Comment:
Since you have 2 nav areas one at the top of the page and one at the bottom, you should target the .footer .nav.  The links at the top of the page are black.  But the footer has a black background so the  text color should be white.  It looks like some how the CSS was removed.